pkg/rollout/rollout_test.go

```diff
@@ -45,6 +45,11 @@ func makeLastRolloutAnnotation(clock clockwork.Clock, offsetFromNowMinute int) s
 	return clock.Now().Add(offset).Format(time.RFC3339)
 }
 
+func addLastUpdateToReport(clock clockwork.Clock, report string) string {
+	report += fmt.Sprintf("\nlastUpdate: %s", clock.Now().Format(time.RFC3339))
```

Comment:
This method seems redundant. You can probably just check existence of lastUpdate: in tests. Or just hardcode the value you expect?